lua/hop/init.lua

```diff
@@ -119,7 +119,18 @@ local function hint_with(hint_mode, opts)
   })
 
   hint.set_hint_extmarks(hl_ns, hints)
-  keymap.create_jump_keymap(0, opts)
+
+  while h == nil do
+    local key = vim.fn.getchar()
+    if type(key) == 'number' then key = vim.fn.nr2char(key) end
```

Comment:
Do we really need this?

Reply:
What exactly, the convoluted check if the getchar result is a number? I think so. While writing this PR, I encountered bugs with the handling of special keys until I sat down, read through `:h getchar()` for the umpteenth time and then did it like this…

Reply:
I just realized I had a bug in this, will fix. Gimme a second

Reply:
Fixed. Multibyte characters are fun

Reply:
They are the worst. Thanks for taking the time to fix this.